app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="white">#FFFFFFFF</color>
+    <color name="cardinal_red">#FF8C1515</color>
+    <color name="cardinal_red_dark">#FF820000</color>
+    <color name="cardinal_red_light">#FFB83A4B</color>
+    <color name="cool_grey">#FF53565A</color>
+    <color name="black_100">#FF2E2D29</color>
```

Comment:
as far as i understand @PSchmiedmayer , we have freedom and it makes sense to stay as close as possible to the material so that we have as little work as possible, we can and should of course also orientate ourselves on the iOS version of the Spezi Kit. for now i just took the colors from here:
https://identity.stanford.edu/design-elements/color/primary-colors/